be/src/exec/vectorized/schema_scanner/schema_charsets_scanner.cpp

```diff
@@ -0,0 +1,73 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021 StarRocks Limited.
+
+#include "exec/vectorized/schema_scanner/schema_charsets_scanner.h"
+
+#include "column/chunk.h"
+#include "exec/vectorized/schema_scanner/schema_helper.h"
+#include "runtime/primitive_type.h"
+#include "runtime/string_value.h"
+
+namespace starrocks::vectorized {
+
+SchemaScanner::ColumnDesc SchemaCharsetsScanner::_s_css_columns[] = {
+        //   name,       type,          size
+        {"CHARACTER_SET_NAME", TYPE_VARCHAR, sizeof(StringValue), false},
```

Comment:
The size looks useless?

Reply:
the whole class is just used for meta info lookup now, it'ok to keep it.